WordPress/Classes/ViewRelated/Gutenberg/GutenbergWeb/GutenbergWebViewController.swift

```diff
@@ -82,6 +82,12 @@ class GutenbergWebViewController: GutenbergWebSingleBlockViewController, WebKitA
         ].compactMap { $0 }
     }
 
+    override func onGutenbergReadyStyles() -> [WKUserScript] {
```

Comment:
Just wanted to flag [this comment from the Gutenberg PR](https://github.com/WordPress/gutenberg/pull/37221#discussion_r771089838) about the function name.

Reply:
Thank you, changed in https://github.com/wordpress-mobile/WordPress-iOS/pull/17644/commits/9a2cf3b0ef36555a72a911a0d3a446187741b062. Let me know if you have further thoughts on the naming, though!